functions.js

```diff
@@ -336,35 +352,33 @@ function saveNewData(id){
 	
 	//edit the display with the new info
 	var infoDisplay = document.getElementById('currentObjects').childNodes;
-	var index = indexToEdit + 1; //offset by 1st text node in the above childNodes
+	var index = indexToEdit; //offset by 1st text node in the above childNodes
 	var infoDisplayCounter = 0;
 	
 	//edit the object in JSONarray
-	for(var i = 0; i < children.length; i++){
-		
-		if(children[i].className === "pair"){			
-			var key;
-			var keyValue;
-			for(var j = 0; j < children[i].childNodes.length; j++){
-				if(children[i].childNodes[j].className === "keyArea"){
+	for (var i = 0; i < children.length; i++) {
+		var key;
+		var value;
+		if (children[i].className === "aside__pair pair") {
+			for (var j = 0; j < children[i].childNodes.length; j++) {
+				if (children[i].childNodes[j].className === "keyArea") {
 					//get key
-					if(i === 1){
-						key = (children[i].childNodes[j].childNodes[3]).value;
-					}else{
-						key = (children[i].childNodes[j].childNodes[1]).value;
-					}
-				}else if(children[i].childNodes[j].className === "value"){
+					console.log("Imong mama");
```

Comment:
do you think we can remove this? 😆

Reply:
## Redundant console. Logs are now removed. 

Console.log was used to determine whether the function was accessed and would display "Imong mama" (means "your mom" in our country 🇵🇭) when the if-statement passed. I used it as a flag to check whether the statements had a bug or not.